nixos/modules/virtualisation/proxmox-image.nix

```diff
@@ -127,8 +127,13 @@ with lib;
       name = "proxmox-${cfg.filenameSuffix}";
       postVM = let
         # Build qemu with PVE's patch that adds support for the VMA format
-        vma = pkgs.qemu_kvm.overrideAttrs ( super: rec {
+        vma = pkgs.qemu_test.overrideAttrs ( super: rec {
```

Comment:
I am not sure if it is a good idea to use the qemu for testing outside of testing

Reply:
Replaced with qemu-kvm.override